src/base/tests/test_util.py

```diff
@@ -1597,15 +1597,16 @@ def test_m2m_no_conflict(self):
         mapping = {g1.id: g3.id, g2.id: g3.id}
 
         u1 = self.env["res.users"].create({"login": "U1", "name": "U1"})
-        u1.groups_id = g1 | g3
-        self.assertEqual(u1.groups_id.ids, [g1.id, g3.id])
+        groups = "group_ids" if util.version_gte("saas~18.1") else "groups_id"
```

Comment:
```suggestion
        groups = "group_ids" if util.version_gte("saas~18.2") else "groups_id"
```

Reply:
Oh, damn, I forgot one again... sorry and thanks :)